web-app/src/components/BookingModal/container.js

```diff
@@ -10,13 +10,24 @@ import {
   getBookingDuration,
 } from '../../reducers';
 import {
+  getAllHolidays,
   updateHoliday,
   requestHoliday,
   rejectionResponse,
   cancelHoliday,
 } from '../../services/holidayService';
 import { requestWFH } from '../../services/wfhService';
 import eventTypes from '../../utilities/eventTypes';
+const testMessageArray = [{ author: 'Paul Tummon', eventMessageId: 1, lastModified: '2018-09-19', message: 'Sorry Spiderman, No go', messageTypeDescription: 'General', messageTypeId: 1 }
+  , { author: 'Spider Man', eventMessageId: 6, lastModified: '2018-09-19', message: 'But Why???', messageTypeDescription: 'General', messageTypeId: 1 },
+{ author: 'Spider Man', eventMessageId: 6, lastModified: '2018-09-19', message: 'But Why???', messageTypeDescription: 'General', messageTypeId: 1 },
+{ author: 'Spider Man', eventMessageId: 6, lastModified: '2018-09-19', message: 'But Why???', messageTypeDescription: 'General', messageTypeId: 1 },
+{ author: 'Spider Man', eventMessageId: 6, lastModified: '2018-09-19', message: 'But Why???', messageTypeDescription: 'General', messageTypeId: 1 },
+{ author: 'Paul Tummon', eventMessageId: 1, lastModified: '2018-09-19', message: 'Sorry Spiderman, No go dsjfdfskgfdskjfhdkjshfsdkfhskdfhdask', messageTypeDescription: 'General', messageTypeId: 1 },
+{ author: 'Paul Tummon', eventMessageId: 1, lastModified: '2018-09-19', message: 'Sorry Spiderman, No go', messageTypeDescription: 'General', messageTypeId: 1 },
+{ author: 'Paul Tummon', eventMessageId: 1, lastModified: '2018-09-19', message: 'Sorry Spiderman, No go', messageTypeDescription: 'General', messageTypeId: 1 },
+{ author: 'Paul Tummon', eventMessageId: 1, lastModified: '2018-09-19', message: 'Sorry Spiderman, No go', messageTypeDescription: 'General', messageTypeId: 1 }];
+
 
```

Comment:
This is hard coded test data as the endpoint isn't as yet feeding back the legacy messages.